capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -12,6 +12,31 @@ const DOWN_KEY = 'ArrowDown';
 const ENTER_KEY = 'Enter';
 let selectedSuggestionPosition = -1;
 
+/**
+ * Gets suggestions to display to the users
+ */
+function getSuggestions() {
+  const fetchStr = '/search?query=' + SEARCH_INPUT.value;
+  fetch(fetchStr).then((response) => response.json()).then((suggestions) => {
+    giveSuggestions(suggestions);
+  });
+}
+
+/**
+ * Sends final college list to the dashboard
+ */
+function sendCollegeInformation() {
+  const listItems = document.querySelector('ul').children;
+  const listArray = Array.from(listItems);
+  let urlHtml = 'dashboard.html?id=';
+  for (const elt of listArray) {
+    urlHtml += elt.id;
+    urlHtml += ',';
+  }
+  urlHtml = urlHtml.slice(0, -1);
+  window.location.href = urlHtml;
```

Comment:
Should we redirect to dashboard if listArray is empty? No action items for this PR. Something to think about.